packages/pypy3.rb

```diff
@@ -0,0 +1,46 @@
+require 'package'
+
+class Pypy3 < Package
+  description 'pypy is fast, drop-in replacement of cpython that can execute code over 4 times faster.'
+  homepage 'https://www.pypy.org/'
+  version '7.3.3'
```

Comment:
Does this version work with python 3.9?

Reply:
this version is python 3.7 i believe
pypy doesn't have python 3.8 or 3.9